tests/additional_seeds/additional_seeds_suite_test.go

```diff
@@ -280,7 +280,7 @@ var _ = Describe(testName, func() {
 			json := `
 			{
 				"spec": {
-					"additionalSeeds": ["192.168.1.1"]
+					"additionalSeeds": ["192.168.1.1", "www.datastax.com"]
```

Comment:
😄

Reply:
I would also check that there are is at least one more IP getting added as you've added **www.datastax.com**. I believe you @jsanda I swear ! 😬